### ISSUE TITLE
Common Module: Connect-SQL verbose messaging

### ISSUE THREAD

**Issue by kungfu71186** [dsccommunity/SqlServerDsc#1354]
Verbose switch is enabled on the verbose message. I'd prefer if verbose wasn't forced
Yes:
`write-verbose 'Message'`

No:
`write-verbose 'Message' -Verbose`

**Comment by johlju** [comment]
This is a workaround for a bug that the Verbose logs are not outputted from helper functions (in an imported module) when for example running a configuration with `Start-DscConfiguration -Verbose`.

**Comment by johlju** [comment]
I can't find the issue where this was discussed (maybe in the DscResource). Can you confirm that the helper function actually output verbose messages without the `-Verbose`?

**Comment by kungfu71186** [comment]
Interesting. I didn't try to run it like that. I'll verify and see.

**Comment by kungfu71186** [comment]
@johlju I'm going to open this as a separate ticket for investigation, i added them back in for now. Although I thought `CmdletBinding` took care of things like that.

**Comment by johlju** [comment]
> Although I thought CmdletBinding took care of things like that.

So did we, when we discussed it. But it seems to only happen when LCM is running the resources, so the $VerbosePreference is not passed to helper functions in the next level of modules. Another workaround is to call the helper functions with ´-Verbose:$VerbosePreference`, but I thought it was to complicated to remember for contributors. But I can have missed the `-Verbose` parameter in new helper function when I have reviewed. 🙂
